themes/powerline-naked/powerline-naked.base.bash

```diff
@@ -6,7 +6,7 @@ source "${BASH_IT?}/themes/powerline/powerline.base.bash"
 function __powerline_left_segment {
 	local OLD_IFS="${IFS}"
 	IFS="|"
-	local params=($1)
+	local params=("$1")
 	IFS="${OLD_IFS}"
```

Comment:
These three lines obviously try to split `$1` with the separator `|`, so `$1` should not be quoted.

Reply:
yup, I moved to fast obviously.